test/smoke/dapi.js

```diff
@@ -1,10 +1,52 @@
+const { client: jaysonClient } = require('jayson/promise');
+const RpcClient = require('@dashevo/dashd-rpc/promise');
+const getNetworkConfig = require('../../lib/test/getNetworkConfig');
+
+const networkConfig = getNetworkConfig();
+
 describe('DAPI', () => {
-  // getBlockHash height:1
-  it('should respond data from chain');
+  networkConfig.inventory.masternodes.hosts.forEach((nodeName) => {
+    describe(nodeName, () => {
+      const dapiClient = jaysonClient.http({
+        host: networkConfig.inventory._meta.hostvars[nodeName].public_ip,
+        port: 3000,
+      });
+      const config = {
+        protocol: 'http',
+        user: networkConfig.variables.dashd_rpc_user,
+        pass: networkConfig.variables.dashd_rpc_password,
+        host: networkConfig.inventory._meta.hostvars[nodeName].public_ip,
+        port: 20002,
+      };
+      const rpc = new RpcClient(config);
+      // getBlockHash height:1
+      it('should respond data from chain', async () => {
+        const { result } = await dapiClient.request('getBlockHash', { height: 1 });
+        const blockHashRpc = await rpc.getBlockHash(1);
+        expect(result).to.be.equal(blockHashRpc.result);
+      });
+
+      // getBestBlockHeight
+      it('should respond data from insight', async () => {
+        const { result } = await dapiClient.request('getBestBlockHeight', { blockHeight: 1 });
+        expect(result).to.be.an('number');
+        expect(result % 1).to.be.equal(0);
+      });
 
-  // getBestBlockHeight
-  it('should respond data from insight');
+      // getBalance from dapi
+      it('should respond data from insight', async () => {
```

Comment:
Why do we have two duplicate tests/

Reply:
will remove "getBalance from dapi". getBestBlockHeight remains